Script.js

```diff
@@ -0,0 +1,52 @@
+const startButton = document.querySelector('#startGameButton');
+const splashScreen = document.querySelector('#displaySplashInstructionsContainer')
+let avatar = document.getElementById('avatar')
```

Comment:
you should make this into a const - it doesn't mean that you can't change the avatar, it just means it can't be accidentally reassigned to something else